.changeset/metal-jobs-compete.md

```diff
@@ -0,0 +1,19 @@
+---
+'@qwik-ui/headless': minor
+---
+
+feat: add vertical carousel option
+
+- feat: add direction and maxSlideHeight props for adjusting the carousel to be vertical
+
+- chore: add directionSig to context
```

Comment:
for internal use we don't need to make a changeset for it.

Reply:
Interesting, I removed this file, however, maybe, we should clarify this in the contribution guide the cases when the contributor should add changeset file [here](https://github.com/qwikifiers/qwik-ui/blob/main/CONTRIBUTING.md#6-added-a-changeset)